apps/web/test/lib/getSchedule.test.ts

```diff
@@ -991,7 +991,7 @@ describe("getSchedule", () => {
           availableSlotsInTz.push(dayjs(timeObj.time).tz(Timezones["+5:30"]));
         });
       }
-      expect(availableSlotsInTz.filter((slot) => slot.format().startsWith(plus2DateString)).length).toBe(23); // 2 booking per day as limit, only one booking on that
+      expect(availableSlotsInTz.filter((slot) => slot.format().startsWith(plus2DateString)).length).toBe(24); // 2 booking per day as limit, only one booking on that
```

Comment:
We have 24 hourly slots a day: 00:00 first slot, 23:00 last slot

Not sure why this passed before, I will take a look at that